internal/indexmigration/migration.go

```diff
@@ -36,14 +36,16 @@ func Done(paths environment.Paths) (bool, error) {
 
 // Migrate removes the index directory and then clones krew-index to the new default index path.
 func Migrate(paths environment.Paths) error {
-	isMigrated, err := Done(paths)
+	isMigrated, err := done(paths)
 	if err != nil {
 		return errors.Wrap(err, "failed to check if index migration is complete")
 	}
 	if isMigrated {
-		klog.V(2).Infoln("Already migrated")
+		klog.V(2).Infoln("already migrated")
```

Comment:
IMO, log messages can be capitalized and end with a dot. Errors on the other hand should not, because they can be wrapped and then it looks weird.

Reply:
Ah ok I can change it, I misremembered reading that log messages shouldn't be capitalized in addition to error messages.